bom/application/pom.xml

```diff
@@ -145,7 +145,7 @@
         <aws-alexa-sdk.version>2.30.0</aws-alexa-sdk.version>
         <azure-functions-java-library.version>1.3.0</azure-functions-java-library.version>
         <kotlin.version>1.3.72</kotlin.version>
-        <dekorate.version>0.12.6</dekorate.version>
+        <dekorate.version>0.12-SNAPSHOT</dekorate.version>
```

Comment:
Let's not forget to change this one :P

Reply:
Yea. Hopefully https://github.com/dekorateio/dekorate/pull/594 will get some attention :-)

Reply:
I pinged @iocanel on this

Reply:
I'll release later today